### PR TITLE
Add import function, supporting native's copy/paste syntax to WWW.

### PR DESCRIPTION
**Pull request by bedmonds**
This one is not only against dev, it also updates the sliders. \o/

### CODE REVIEW

File: FF1RandomizerOnline/wwwroot/js/site.js [hunk 1]
```diff
@@ -117,6 +121,17 @@ function setCallbacks() {
 	};
 }
 
+function OnFlagsChanged() {
```
Comment (Entroper) [inline FF1RandomizerOnline/wwwroot/js/site.js]: Make this camelCase and we're good to go.

Comment (bedmonds) [reply FF1RandomizerOnline/wwwroot/js/site.js]: We should all agree that kebab-case in a sea of parentheses is the superior choice and be done with it. :D

File: FF1RandomizerOnline/wwwroot/js/site.js [hunk 2]
```diff
@@ -19,6 +19,19 @@ function validateFlags() {
...
+	var str = prompt("Paste in a seed and flags string as given to you by our lord and master, crim_bot. (SEED_FLAGS)");
+	var seed;
+	var flags;
+
+	[seed, flags] = str.split("_", 2);
+
+	document.getElementById("Flags").value = flags;
+	document.getElementById("Seed").value = seed;
+
+	OnFlagsChanged();
```
Comment (Entroper) [inline FF1RandomizerOnline/wwwroot/js/site.js]: D'oh.

Comment (bedmonds) [reply FF1RandomizerOnline/wwwroot/js/site.js]: Nerf 6:30 AM late-night coding.

**Pull request merged**